test/units/modules/cloud/xenserver/test_xenserver_guest_facts_module_main.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
```

Comment:
Does ansible-test actually find this when PRs for xenserver_guest_facts are found? I know that for example for `module_utils/basic.py`, there's a whole subdirectory containing the tests for it: https://github.com/ansible/ansible/tree/devel/test/units/module_utils/basic